script/QA/QA_25.txt

```diff
@@ -60,7 +60,7 @@ I'm sure you already understand this, but...
 [sha:83c0e7146d9d2afd63c389ce5992e6cbb0e505b0]{
 -- Page 322, Offset 43909.
 -- 『アルクェイド・ブリュンスタッドの物語』と、
-The stories of Ciel and Arcueid Brunestud have come to a%{n}
+The stories of Arcueid Brunestud and Ciel have come to a%{n}
```

Comment:
I know they say it the other way around, but for whatever reason this felt like the more natural order to me /shrug

Reply:
~~Felt natural for Arc to be first~~Let's just go with in order of how they are mentioned.